cfgrammar/src/lib/yacc/mod.rs

```diff
@@ -47,7 +47,7 @@ impl quote::ToTokens for YaccKind {
         tokens.extend(match *self {
             YaccKind::Grmtools => quote!(::cfgrammar::yacc::YaccKind::Grmtools),
             YaccKind::Original(action_kind) => {
-                quote!(::cfgrammar::yacc::YaccKind::Original(#action_kind))
+                quote!(::cfgrammar::yacc::YaccKind::Original(#action_kind,))
```

Comment:
I think perhaps it is best to attempt to do something less fragile here.

My thoughts are instead of emitting the cache module, constant string containing the cache module,
in the hopes that the pretty printer will ignore the string.

Reply:
Fixed by [8e32c81](https://github.com/softdevteam/grmtools/pull/525/commits/8e32c81807fd0aab74565f33893eb99517e2a36b) and subsequent commits...